### PR TITLE
Initial: Transparent Cross-Team Decision Making using RFCs

### PR DESCRIPTION
**Pull request by spier**
We created a first draft of the pattern "Transparent Cross-Team Decision Making using RFCs", after an interview between Sebastian Spier and Tom Sadler.

This implements #174.

We would like to get this pattern merged before the InnerSource Summit, so that Tom can point to this pattern as part of his talk "Using internal RFCs to enhance collaboration".

Even though this pattern is already fairly complete, we are submitting it as state **Initial**, so that it can hopefully be merged faster.

**Future:**
After the InnerSource Fall Summit 2020 we plan to extend this pattern with additional info based on Tom Sadler’s talk at the summit. We will send separate PRs for this. With that we then want to bring the pattern to maturity “Structured”.

### CODE REVIEW

**Review by lenucksi — APPROVED** [review]
Content fulfils all expectations of the Contributors Handbook for 1-initial. LGTM from me.
Europace is mentioned, hence mentioning @MaineC.

Happy to merge if there are no further additions from @MaineC.

**Comment by spier** [comment]
Ah. That was short-sighted by me.

@MaineC has likely not seen this yet, and I did not talk with her about this pattern either.

So maybe removing Europace is a faster path to a merge as maturity **Initial**?

**Comment by spier** [comment]
Hi @lenucksi. I spoke to @MaineC yesterday, and I got this confirmation from her:
> If you do not hear back from me until lunch tomorrow, assume it's ok to leave the name there.

So we are good to merge this pattern as **Initial**.

Thanks for your help!

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:27]: Why? (Track decisions for future reference, include people in the decision making process who are not on-site etc.)

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:23]: Another reason could be cross team dependencies where making changes openly can help resolve competing customer/user requirements by making those requirements visible to everyone.

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:25]: Instead of users maybe stakeholders?

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:31]: Why? (Think of a desire for fast feedback as opposed to big design up front.)

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:32]: I'm not sure implementation should be forbidden during that phase - there could even be prototypes that help with the discussion.

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:33]: I would add that there may be a bias for action over coordination leading to less bandwidth for aligning teams.

Maybe add that openly discussing designs leads to more input and in many cases more robust architecture.

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:40]: Would be helpful to share a sample process with the pattern?

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:67]: Typo: s/man development/many development/

File: patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md
Comment (MaineC) [inline patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md:74]: This should go to the Known Instances.

**Review by MaineC — COMMENTED** [review]
Overall the pattern looks really good for an initial stage pattern. A lot of the things I found could be addressed in separate pull requests as well.

I would love to see comments from @NewMexicoKid on the current structure - his insight has helped me  a lot of time to find a good structure for patterns.

**Comment by spier** [comment]
@MaineC thanks for your review of the pattern. Unfortunately I did not see this before we got this particular PR merged.

Would the following approach work for you?

- I work with @tsadler1988 to get his research from the recent Summit worked into this pattern
- we will review your comments above and see how we can work that in
- then we will create a new PR with all of that and ask you for a review

Would that work for you, or can you think of a better approach?

**Comment by MaineC** [comment]
No worries about the missed comments. It's great to have this pattern in the repo. The approach you suggested sounds just fine to me.

**Pull request merged**
